core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 require 'rails/generators'
+require 'rails/generators/app_base'
 require 'rails/version'
 require_relative 'install_generator/bundler_context'
 require_relative 'install_generator/support_solidus_frontend_extraction'
 require_relative 'install_generator/install_frontend'
 
 module Solidus
   # @private
-  class InstallGenerator < Rails::Generators::Base
+  class InstallGenerator < Rails::Generators::AppBase
```

Comment:
@elia Can you review if my analysis of the issue (see PR description) is correct? Thanks!